x-pack/plugins/transform/public/app/common/validators/retention_policy_max_age_validator.test.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { retentionPolicyMaxAgeValidator } from './retention_policy_max_age_validator';
+
+describe('Transform: retentionPolicyMaxAgeValidator()', () => {
+  const transformRetentionPolicyMaxAgeValidator = (arg: string) =>
+    retentionPolicyMaxAgeValidator(arg).length === 0;
```

Comment:
I reckon we should not write unit tests for functions created in the test suites itself

Reply:
Fixed in [0193145](https://github.com/elastic/kibana/pull/173736/commits/01931450a6b508122f7138118dbd61a556d22528).